public/js/components/Editor.js

```diff
@@ -36,7 +36,7 @@ function resizeBreakpointGutter(editor) {
 const Editor = React.createClass({
   propTypes: {
     breakpoints: ImPropTypes.map.isRequired,
-    selectedSource: ImPropTypes.map,
+    location: PropTypes.object,
```

Comment:
I could see `selectedLocation` being slightly clearer. There are many locations in the editor, breakpoint locations, selected gutter lines, ... I don't feel strongly though